pulsar-client-cpp/tests/BackoffTest.cc

```diff
@@ -51,7 +53,7 @@ TEST(BackoffTest, firstBackoffTimerTest) {
     Backoff backoff(milliseconds(100), seconds(60), milliseconds(1900));
     ASSERT_EQ(backoff.next().total_milliseconds(), 100);
     boost::posix_time::ptime firstBackOffTime = PulsarFriend::getFirstBackoffTime(backoff);
-    usleep(300 * 1000);
+    boost::this_thread::sleep(boost::posix_time::milliseconds(300));
```

Comment:
We could also `std::this_thread::sleep_for()` instead of boost

Reply:
That's true and it does mean one less boost lib to link. If that is preferable I can go ahead and make the change.

Reply:
I did go ahead and switch everything to std::this_thread::sleep_for as you suggested

Reply:
Nice. Thanks!